client/packages/common/src/intl/context/IntlContext.tsx

```diff
@@ -22,6 +22,13 @@ export const IntlProvider: FC<PropsWithChildrenOnly> = ({ children }) => {
       : // TODO: change back to a week when things are stable
         60 * minuteInMilliseconds; // 7 * 24 * 60 * minuteInMilliseconds;
 
+    // backend path for loading locale files
+    // electron app: requires a relative path
+    // browser: requires an absolute path
+    const userAgent = navigator.userAgent.toLowerCase();
+    const isElectron = userAgent.indexOf(' electron/') > -1;
```

Comment:
hmm, we can also make a helper in useElectronClient for see i window.electronAPI is there.

Reply:
Although I am not sure if this will work if app navigates to new tab (maybe new windows that are auto opened will not have preload.ts